src/wp-admin/update-core.php

```diff
@@ -1011,11 +1011,30 @@ function do_undismiss_core_update() {
 	$help_sidebar_autoupdates = '<p>' . __( '<a href="https://wordpress.org/documentation/article/plugins-themes-auto-updates/">Documentation on Auto-updates</a>' ) . '</p>';
 }
 
+$help_sidebar_rollback = '';
+
+if ( current_user_can( 'update_themes' ) || current_user_can( 'update_plugins' ) ) {
+	$rollback_help = '<p>' . __( 'Rollback is a feature that will create a temporary backup of a plugin or theme before it is upgraded. This backup is used to restore the plugin or theme back to its previous state if there is an error during the update process.' ) . '</p>';
+
+	$rollback_help .= '<p>' . __( 'On systems with fewer resources, this may lead to server timeouts or resource limits being reached. If you encounter an issue during the update process, please create a support forum ticket and reference <strong>Rollback</strong> in the issue title.' ) . '</p>';
+
+	get_current_screen()->add_help_tab(
+		array(
+			'id'      => 'rollback-plugins-themes',
+			'title'   => __( 'Rollback' ),
```

Comment:
As title for the tab, instead of `Rollback`, what about `Automatic reinstall after failed update`. A bit long I know, but it descibes the feature.

Reply:
I think it's a bit too long… what about "Restoration"

Reply:
That sound better to me. Is the addition of plugin/theme to much for the string? Like: `Plugin & Theme restoration`, `Plugin and theme restoration` or `Restoration for themes and plugins`.

Just a mockup image
![bild](https://github.com/WordPress/wordpress-develop/assets/11491369/092dffa5-c444-45e7-9098-bdbd9b491054)

Reply:
Is there something wrong or incorrect with `Rollback` here?